pkg/tests/fixtures.go

```diff
@@ -8,14 +8,39 @@ package tests
 import (
 	"fmt"
 
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/annotations"
 	n "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
 	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/golang/glog"
 	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+const (
+	fixturesNamespace     = "--namespace--"
```

Comment:
I'd also remove the `fixtures...` prefix.
Me probably are going to need these in other modules - so I'd also capitalize them so they are exported.
Also I'd remove the `const` block in `test_fixtures.go` and reuse the constants from this block here.